.dockerignore

```
@@ -0,0 +1,8 @@
+.git
+.env
+.vscode
+.venv
+.pytest_cache
+__pycache__
```

```suggestion
**/__pycache__
```
Annoying, but worth noting that `.dockerignore` has slightly different behaviour to `.gitignore`, in particular in that it _doesn't_ ignore subfolders by default - we need to explicitly reference them with the `**` prefix to mean arbitrary levels of nesting should be considered for this rule. I think at a glance that this is the only entry that matters here, but this is one of those issues that can easily lead to weird confusion when some behaviour starts caching oddly!